OOStepByStepTest/UnitTest1.cs

```diff
@@ -6,10 +6,58 @@ namespace OOStepByStepTest
     public class UnitTest1
     {
         [Fact]
-        public void Test1()
+        public void Should_person_can_introduce_himself()
         {
-            var class1 = new Class1();
-            Assert.NotNull(class1);
+            // given
+            var person = new Person("Tom", 21);
+            // when
+            var msg = person.Introduce();
+            // then
+            Assert.Equal("My name is Tom. I am 21 years old", msg);
+        }
+
+        [Fact]
+        public void Should_student_can_introduce_himself()
```

Comment:
You can name it as:
Should_..._when_..._given_...

Reply:
got it, thanks~